### PR TITLE
Comments for blog posts

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

This change includes comments for blog posts, behind a feature flag. The feature flag is a requirement as Discourse only allows embedding on pre-set domains for security reasons. With a feature flag we can hide this feature for all visitors until we want to remove the flag.

Further this change links the support forum from the resources nav item.

### CODE REVIEW

**Comment by joepavitt** [comment]
hubspot form is being rendered oddly:

Screenshot 2023-05-30 at 12 12 38

**Comment by ZJvandeWeg** [comment]
> hubspot form is being rendered oddly, can be seen int he deploy preview:
> 
> Screenshot 2023-05-30 at 12 12 38

Did you fix this through: 966e4af540d3054d4d347421b05d856562dc2393?

File: src/_includes/layouts/post.njk [hunk 1]
```diff
@@ -40,25 +40,25 @@ layout: layouts/base.njk
...
-                    (function () {
-                        var d = document.createElement('script');
-                        d.type = 'text/javascript';
-                        d.async = true;
-                        d.src = window.DiscourseEmbed.discourseUrl + 'javascripts/embed.js';
-                        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(d);
-                    })();
-                </script>
-                  {% endabtesting %}
+                        <script type="text/javascript">
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply src/_includes/layouts/post.njk:48]: not sure I follow? The sign up form positions incorrectly now, independent of the abtesting workflow.

**Comment by joepavitt** [comment]
> > hubspot form is being rendered oddly, can be seen int he deploy preview:
> > Screenshot 2023-05-30 at 12 12 38
> 
> Did you fix this through: [966e4af](https://github.com/flowforge/website/commit/966e4af540d3054d4d347421b05d856562dc2393)?

nope

**Comment by joepavitt** [comment]
currently trying to understand why the edge function isn't completing on Netlify deploy preview :(

**Comment by joepavitt** [comment]
With https://github.com/flowforge/website/pull/794 merged, this can now be rebased in order to have functional edge functions

**Comment by ZJvandeWeg** [comment]
@joepavitt The merge does get the fix in too, as the main branch' tip will be the parent of this merge commit. But I've rebased and will merge this PR once the tests succeeded

**Comment by ZJvandeWeg** [comment]
Fixed the broken link on the main branch directly, as the failure was observed there too

**Pull request merged**